string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,27 @@
+/* exported isVowel */
+function isVowel(char) {
+  switch (char) {
+    case 'a':
+      return true;
+    case 'A':
+      return true;
+    case 'e':
+      return true;
+    case 'E':
+      return true;
+    case 'i':
+      return true;
+    case 'I':
+      return true;
+    case 'O':
+      return true;
+    case 'o':
+      return true;
+    case 'U':
+      return true;
+    case 'u':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
If we convert `char` to lowercase, we can simplify this. That way we only need to check for lowercase values.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```

Reply:
if we use .toLowerCase(), how does it check for the uppercase? Also, do I not need a return statement after each case??